resources/production/chainspec.toml

```diff
@@ -173,6 +173,10 @@ maximum_round_length = '66 seconds'
 # Expressed as a fraction (1/5 by default).
 reduced_reward_multiplier = [1, 5]
 
+[highway.performance_meter]
+# The number of recent blocks to consider when measuring performance for the purpose of deciding the round length.
+blocks_to_consider = 10
+
```

Comment:
Same as above - old thing, don't reintroduce.

Reply:
Removed in 080a969198464c157308f2b8dffd72450c0aece4